package.json

```diff
@@ -35,6 +35,7 @@
     "prettier": "^3.2.5",
     "prettier-plugin-tailwindcss": "^0.6.2",
     "tailwindcss": "^3.4.4",
+    "tailwindcss-textshadow": "^2.1.3",
```

Comment:
Looking at the npm, it looks like it's a pretty old plugin, it might be incompatible with the latest versions of Tailwind or other dependencies

https://www.npmjs.com/package/tailwindcss-textshadow

Reply:
if it works properly without error I will leave it

Reply:
Old but gold. Works like a Swiss watch.